s3/pkg/messagebus/tests/message_bus_suite_test.go

```diff
@@ -5,8 +5,8 @@ import (
 	"testing"
 
 	"github.com/opensds/multi-cloud/s3/pkg/helper"
-	"github.com/opensds/multi-cloud/s3/pkg/log"
-	bus "github.com/opensds/multi-cloud/s3/pkg/messagebus"
+	log "github.com/sirupsen/logrus"
+        bus "github.com/opensds/multi-cloud/s3/pkg/messagebus"
```

Comment:
Plz format this file.

Reply:
Log references are missing. I am reverting back changes according to development branch.
I have opened an issue for tracking this.
https://github.com/sodafoundation/multi-cloud/issues/975